### ISSUE TITLE
console.google.com docs and this README disagree

### ISSUE THREAD

**Issue by ekalosak** [google-github-actions/setup-gcloud#651]
### TL;DR

On [console.google.com](https://cloud.google.com/iam/docs/reference/sts/rest/v1/TopLevel/token), it says:
```
audience | string                     The full resource name of the identity provider; for example: //iam.googleapis.com/projects/<project-number>/locations/global/workloadIdentityPools/<pool-id>/providers/<provider-id> for workload identity pool providers, or //iam.googleapis.com/locations/global/workforcePools/<pool-id>/providers/<provider-id> for workforce pool providers. Required when exchanging an external credential for a Google access token.
```
That is, it _includes_ the `//iam.googleapis.com/` prefix.

In this repo's README, it says:
```
    steps:
    - id: 'auth'
      uses: 'google-github-actions/auth@v1'
      with:
        workload_identity_provider: 'projects/123456789/locations/global/workloadIdentityPools/my-pool/providers/my-provider'
        service_account: 'my-service-account@my-project.iam.gserviceaccount.com'
```
That is, it _drops_ the domain from the URI.

tl;dr Google's developer docs are not in sync with each other.

### Action YAML

```yaml
.
```
